dask_expr/_groupby.py

```diff
@@ -753,6 +753,30 @@ def grp_func(self):
         return functools.partial(_groupby_slice_transform, func=self.func)
 
 
+def _fillna(group, *, what, **kwargs):
+    return getattr(group, what)(**kwargs)
+
+
+class GroupByBFill(GroupByTransform):
+    func = staticmethod(functools.partial(_fillna, what="bfill"))
+
+    def _simplify_up(self, parent):
+        if isinstance(parent, Projection):
```

Comment:
if I replace this line with `if True:` all tests remain green

Reply:
I am not too worried about this, we can add a general test for all groupby methods in a follow up though